### PR TITLE
Removes label `experimental`  of ember-engines

### PR DESCRIPTION
**Pull request by villander**
Update README following status reported by @dgeb - https://discuss.emberjs.com/t/readers-questions-what-is-the-status-of-ember-engines/14236

@rwjblue @dgeb we need replace `Experimental support for the Ember Engines RFC ` for `Composable applications for ambitious user experiences`

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ perspective.
 
 ## Important Note about Compatibility and Stability
```
Comment (rwjblue) [inline README.md]: Lets remove this section completely

Comment (villander) [reply README.md]: done!

**Comment by villander** [comment]
done @rwjblue, we can merge it now ; )

**Comment by villander** [comment]
done @rwjblue, we can merge it now ; )

**Comment by villander** [comment]
@rwjblue we still need replace `Experimental support for the Ember Engines RFC` for `Composable Ember applications for ambitious user experiences` 

I don't have permission to do it.

**Comment by rwjblue** [comment]
done

**Comment by villander** [comment]
thanks ; )

**Pull request merged**